.github/workflows/ci.yaml

```diff
@@ -101,10 +105,17 @@ jobs:
             cargo-cache-
 
       - name: Run command
+        if: ${{ github.ref != 'refs/heads/main' }}
```

Comment:
(nit) you don't need the ${{ here (in fact I run into some issues with that at some point, I don't remember why)

Reply:
Interesting! I copied this from the [github reference](https://docs.github.com/en/actions/learn-github-actions/contexts#determining-when-to-use-contexts)!